compose/cli/main.py

```diff
@@ -486,8 +486,12 @@ def rm(self, options):
         Options:
             -f, --force   Don't ask to confirm removal
             -v            Remove volumes associated with containers
+            -a, --all     Also remove one-off containers
```

Comment:
I wonder if users know what "one-off" containers are, or if we should add "one-off containers created by docker-compose run".